### PR TITLE
Remove manual changes from bundle/manifests

### PR DESCRIPTION
**Pull request by jlpadilla**
### Related Issue
N/A

### Description of changes
- Move manual changes merged into bundle/manifests to corresponding source files so it can be re-generated.
- Updated README with Development notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,92 +4,80 @@ Deploys the Open Cluster Management Search v2 components.
 
 ## Installing the Search Operator in a Red Hat OpenShift cluster
 
-### 0. Prerequisites
```
Comment (jlpadilla) [inline README.md:7]: NOTE: The only addition is the **##Development** section. All other changes are formatting.

File: README.md [hunk 2]
```diff
@@ -101,119 +89,134 @@ cd ./scripts
...
-```
-
-#### 3. Update the operator version in bundle clusterserviceversion to use the image built in step (2)
-
-```bash
-Update the file search-v2-operator/bundle/manifests/search-v2-operator.clusterserviceversion.yaml in the container named manager to use the image above.
-```
-
-### **Building search-v2-operator-bundle in your cluster:**
+## Development
```
Comment (jlpadilla) [inline README.md:92]: **Note to reviewers:** here's the added section. Goes through line 110.

File: bundle/manifests/search-v2-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -194,7 +194,10 @@ spec:
           - create
         serviceAccountName: search-v2-operator
       deployments:
-      - name: search-v2-operator-controller-manager
+      - label:
```
Comment (jlpadilla) [inline bundle/manifests/search-v2-operator.clusterserviceversion.yaml:197]: **Note to reviewers:** Added label shouldn't be a problem. Also I added the `app=search` label.

File: bundle/manifests/search-v2-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -222,7 +226,7 @@ spec:
                 resources:
                   limits:
                     cpu: 200m
-                    memory: 1024Mi
+                    memory: 1Gi
```
Comment (jlpadilla) [inline bundle/manifests/search-v2-operator.clusterserviceversion.yaml:229]: **Note to reviewers:** Unit conversion was done automatically, it shouldn't cause a problem.

File: bundle/manifests/search-v2-operator.clusterserviceversion.yaml [hunk 5]
```diff
@@ -232,14 +236,6 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/stolostron/search-v2-operator:2.7.0-dc0fc8c59239b6a0a3f440b7752101523d87f597
```
Comment (jlpadilla) [inline bundle/manifests/search-v2-operator.clusterserviceversion.yaml:235]: **Note to reviewers:** This section got moved below, contents are exactly the same.

File: README.md [hunk 6]
```diff
@@ -101,119 +89,134 @@ cd ./scripts
...
+- Use `make manifests` and `make bundle` to propagate your changes into the bundle directory.
+- Validate the generated bundle manifest files with:
+  ```bash
+  oc apply -f bundle/manifests --dry-run=server 
+  ```
+- Update the CRDs in your target cluster with `make install`.
 
-#### 5. Create the Image pull secret to pull image from quay
+**Committing changes:**
+- Run `make genreate; make manifests; make bundle` before committing changes.
```
Comment (SherinV) [inline README.md:110]:
```suggestion
- Run `make generate; make manifests; make bundle` before committing changes.
```

File: Makefile [hunk 7]
```diff
@@ -36,7 +36,7 @@ IMAGE_TAG_BASE ?= open-cluster-management.io/search-v2-operator
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/stolostron/search-v2-operator:latest
+IMG ?= quay.io/stolostron/search-v2-operator:2.7.0-dc0fc8c59239b6a0a3f440b7752101523d87f597
```
Comment (SherinV) [inline Makefile:39]: why 2.7 image?

Comment (jlpadilla) [reply Makefile:39]: I used the value that's currently in the csv at bundle/manifests to reduce the changes to the csv.

These get overwritten later by installer.
Maybe we could change it to something like `quay.io/stolostron/search-v2-operator:tag-placeholder`

File: README.md (hunk 6 above)
Comment (jlpadilla) [reply README.md]: done

File: Makefile (hunk 7 above)
Comment (SherinV) [reply Makefile:39]: ok, thanks.

**Review by SherinV — APPROVED** [review]
/lgtm
/hold

File: config/manager/manager.yaml [hunk 8]
```diff
@@ -48,9 +63,11 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 100Mi
+            memory: 1024Mi
           requests:
-            cpu: 100m
-            memory: 20Mi
+            cpu: 50m
```
Comment (SherinV) [inline config/manager/manager.yaml:68]: These values changed. Does it matter?

Comment (jlpadilla) [reply config/manager/manager.yaml:68]: Changed to match the values we have on /bundle/manifests/ because `make bundle` was changing it back to these.

**Review by SherinV — APPROVED** [review]
/lgtm
/hold

**Pull request merged**